### PR TITLE
DEVEXP-545 Can now test connections

### PR DESCRIPTION
**Pull request by rjrudin**
I am going to add some more tests to this - unit tests to verify that the messages are built correctly - but wanted to get this out for review since I think the app code is ready to go.

### CODE REVIEW

File: src/main/java/com/marklogic/appdeployer/command/TestConnectionsCommand.java [hunk 1]
```diff
@@ -0,0 +1,295 @@
...
+	 */
+	public TestResults testConnections(CommandContext context) {
+		TestResult manageResult = testManageAppServer(context.getManageClient());
+		TestResult adminResult = testAdminAppServer(context.getAdminManager());
+
+		TestResult appServicesResult = null;
+		TestResult restResult = null;
+		TestResult testRestResult = null;
+
+		if (manageResult.isSucceeded()) {
```
Comment (rjrudin) [inline src/main/java/com/marklogic/appdeployer/command/TestConnectionsCommand.java:67]: Yuck, this is a long method! Going to shorten this too.

File: src/main/java/com/marklogic/mgmt/admin/AdminManager.java [hunk 2]
```diff
@@ -21,7 +21,12 @@
 import com.marklogic.rest.util.RestTemplateUtil;
 import org.springframework.core.io.ByteArrayResource;
 import org.springframework.core.io.Resource;
-import org.springframework.http.*;
+import org.springframework.http.HttpEntity;
```
Comment (rjrudin) [inline src/main/java/com/marklogic/mgmt/admin/AdminManager.java:24]: Ick, I'm going to unformat this to remove all the whitespace changes.

**Review by BillFarber — APPROVED** [review]
